apps/test/unit/code-studio/components/libraries/LibraryCreationDialogTest.js

```diff
@@ -1,4 +1,4 @@
-import {expect} from '../../../../util/configuredChai';
+import {expect} from '../../../../util/deprecatedChai';
```

Comment:
This is unrelated to my changes but my Drone runs were failing because the import couldn't be found.  I updated based on https://github.com/code-dot-org/code-dot-org/pull/31295